includes/today-main-site-feed-options.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Provides functions for the Main Site Story feed
+ */
+
+function tu_add_main_site_feed_options_page() {
+	if ( function_exists( 'acf_add_options_page' ) ) {
+		acf_add_options_page( array(
+			'page_title' 	  => 'Main Site News Feed',
+			'post_id'         => 'main_site_news_feed',
+			'menu_title'	  => 'EDU News Feed',
+			'menu_slug' 	  => 'main-site-news-feed',
+			'capability'	  => 'administrator',
+			'icon_url'        => 'dashicons-images-alt2',
+			'redirect'        => false,
+			'updated_message' => 'Main Site News Feed Updated'
+		) );
+	}
+}
+
+function tu_post_get_thumbnail( $object, $field_name, $request ) {
```

Comment:
In a future update we'll want to update how this logic works, since we're no longer using the featured image field on posts.  (Maybe just use `today_get_thumbnail_id()` from the child theme instead?)